lib/models/toponyme.js

```diff
@@ -4,6 +4,8 @@ const {validPayload} = require('../util/payload')
 const {getFilteredPayload} = require('../util/payload')
 const position = require('./position')
 const {cleanNom} = require('./voie')
+const nomAltSchema = require('./nom-alt')
+const {getNomAltDefault} = require('./nom-alt')
```

Comment:
```suggestion
const nomAltSchema, {getNomAltDefault) = require('./nom-alt')
```

ou alors juste import `nomAltSchema ` et faire `nomAltSchema. getNomAltDefault `